dashboard/lib/services/user/password_resetter_by_email.rb

```diff
@@ -45,6 +45,11 @@ def call
       private def send_reset_password_instructions
         raw = user.send(:set_reset_password_token)
         user.send(:send_devise_notification, :reset_password_instructions, raw, {to: email})
+        Cdo::Metrics.put(
+          'User', 'PasswordResetEmailSuccessful', 1, {
+            Environment: CDO.rack_env
```

Comment:
Since the metrics are related to the User namespace, shouldn't we include the user ID, for instance, to differentiate these metrics for each user?
```suggestion
            Environment: CDO.rack_env,
            UserID: user.id,
```

Reply:
Adding the user id creates a new dimension for the log. This makes it a bit harder to find the total numbers for these metrics. We're not necessarily interested in who is getting password resets, just how many users are requesting password resets (and comparing successful resets vs no user found vs no email auth option found).